src/Form/Type/PayzenGatewayConfigurationType.php

```diff
@@ -36,6 +36,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'sylius.form.gateway_configuration.payzen.ctx_mode.choices.production' => Api::MODE_PRODUCTION,
                 ),
             ])
+            ->add('endpoint', ChoiceType::class, [
+                'label'       => 'sylius.form.gateway_configuration.payzen.endpoint.label',
+                'choices'  => array(
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.default' => null,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.clic_and_pay' => Api::ENDPOINT_CLICANDPAY,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.scellius' => Api::ENDPOINT_SCELLIUS,
+                    'sylius.form.gateway_configuration.payzen.endpoint.choices.systempay' => Api::ENDPOINT_SYSTEMPAY,
+                ),
```

Comment:
@FlorentinGarnier can you add those missing translations into the `src/Resources/translations/messages.*.yaml`

Reply:
Sorry some files was not added in my last commit :)